MyApp/app/controllers/users/sessions_controller.rb

```diff
@@ -2,6 +2,8 @@
 
 class Users::SessionsController < Devise::SessionsController
   # before_action :configure_sign_in_params, only: [:create]
+  before_action :set_last_seen_at, if: proc { current_user && (session[:last_seen_at] == nil || session[:last_seen_at] < 15.minutes.ago) }
```

Comment:
@olga-kolosek Not bad. Samo napisi pretragu na odgovarajucem mestu. Rails scopes pogledaj.

Reply:
Ostavila sam before_action, ali samo za sign out funkciju

Reply:
@olga-kolosek Rails scopes. To treba implementirati, before_action ali i scope.